pkg/querier/ingester_querier.go

```diff
@@ -35,6 +35,10 @@ import (
 	util_log "github.com/grafana/loki/v3/pkg/util/log"
 )
 
+var defaultQuorumConfig = ring.DoUntilQuorumConfig{
+	// Nothing here
```

Comment:
I think redundant comment. If we do want a comment I would write something like this (finishing the sentence):

```suggestion
// We use the usable, default value of `DoUntilQuorumConfig` as ...
var defaultQuorumConfig = ring.DoUntilQuorumConfig{}
```